css/style.css

```diff
@@ -1,11 +1,217 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
```

Comment:
I would remove development comments like this as they will be visible in the source code in the browser